.gitignore

```diff
@@ -102,3 +102,12 @@ venv.bak/
 
 # mypy
 .mypy_cache/
+
+# VSCode
+.vscode
+
+# SonarLint
+.sonarlint
+
+# Downloaded Media
+media/
```

Comment:
Maybe you can erase this part, because this is a really good option and i really want to use it. Thanks both!